src/namespaces/languages.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (io) => {
+    const languages = io.of('/languages');
```

Comment:
Just a domain channel, a chat room.

Reply:
Should we name it "/chat-room" or something like that?

Reply:
This represents an item in the channel list (random, general, speakers...), to don't get confuse I will change the name to random-channel (align with wiki). I prefer channel subfix because on socket.io the concept does  not exist.

Reply:
cool now I think io get it, thanks for the clarification